.eslintrc.yml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
This is a Node (JavaScript) project, so let's use JSON for this.

Reply:
Sure, but lets use `.eslintrc.js` for including comments 😉